src/patchkernel/patch_kernel.cpp

```diff
@@ -5635,6 +5635,147 @@ void PatchKernel::getInterfaceVertexCoordinates(long id, std::array<double, 3> *
 	getElementVertexCoordinates(interface, coordinates);
 }
 
+/*!
+   Check if an interface  is intersected by a given plane. Calculate the intersection
+   as a segment defined by two points and the part of the interface belonging to the subspace
+   opposite to the one pointed by the given normal. If the interface is not intersected,
+   an empty polygon is returned. 
+
+   Interfaces laid on the plane are not considered as intersected ones.
+
+   Interfaces having an edge on the plane are not considered as intersected ones.
+
+   \param[in] P0 First triangle vertex
+   \param[in] P1 Second triangle vertex
+   \param[in] P2 Third triangle vertex
```

Comment:
The documentation doesn't match the argument of the function.

Reply:
Done